planet/planet.go

```diff
@@ -461,3 +468,11 @@ func injectAssetIntoMetadata(feature *geojson.Feature, asset Asset) {
 		feature.Properties["type"] = asset.Type
 	}
 }
+
+func EvalDateParam(context *Context, date string, field string) (eval bool, err error) {
+	if eval, _ := regexp.MatchString("^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}(\\.[0-9]+)?Z", date); !eval {
```

Comment:
Have you considered using `time.Parse` instead of a manual regular expression?

```go
if _, err := time.Parse("2006-01-02T15:04:05Z", date); err != nil {
  // Format is wrong!
}
```

It would be more readable and uses a more specialized module that may do extra validation (e.g. making sure that nobody passes in "42" for the day of month)